sympy/solvers/solvers.py

```diff
@@ -382,77 +382,107 @@ def solve(f, *symbols, **flags):
         - piecewise combinations of the above
         - systems of linear and polynomial equations
         - systems containing relational expressions
+        - systems implied by undetermined coefficients
 
     Examples
     ========
 
-    The output varies according to the input and can be seen by example:
+    The default output varies according to the input and might
+    be a list (possibly empty), a dictionary, a list of
+    dictionaries or tuples, or an expression involving relationals.
+    An document describing the conditions under which different forms
+    of output are given is in explanations section of the documentation.
```

Comment:
There should be a clickable link here in the html docs. Just needs a reference to the `solve_output` anchor.